core/src/main/scala/scalaz/Show.scala

```diff
@@ -6,7 +6,7 @@ package scalaz
  * [[scalaz.Cord]] for efficiency.
  */
 ////
-trait Show[F]  { self =>
+trait Show[F] extends Show.ContravariantShow[F]  { self =>
```

Comment:
Why is `ContravariantShow` needed?

Reply:
Without that it infers type `Result1T` not `CopructT` and cannot reach the Shows instance. Should only apply in the string interpolator context. As `Show` is still invariant.

Reply:
I would say that's an orthogonal problem, not limited to just `Show`. Don't try to solve it in this PR.

Reply:
So leave test pending and remove the `ContravariantShow`?

Reply:
Remove `ContravariantShow`. Either remove the offending test, or add a type annotation: `val r: CoproductT = Result1T`.